README.md

```diff
@@ -78,3 +84,7 @@ enable = true
 bind = "127.0.0.1"
 port = 9090
 ```
+
+## Terraform
+
+Implement Triton Service Groups in Terraform by using a template resource.
```

Comment:
This part is incomplete, yes? It's both the template and the service group resource, I think...?

Let's cut this and leave it to others on the team to detail.